ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1486,6 +1504,24 @@ setOptions({
         }
     }
 });
+addEvent(H.Chart, 'render', function (): void {
+    const chart = this,
+        stockTools = chart.stockTools,
+        button = chart.stockTools &&
+            chart.stockTools.toolbar &&
+            chart.stockTools.toolbar.querySelector('.highcharts-current-price-indicator');
+
+    // Change the initial button background.
+    if (stockTools && chart.options.series && button) {
+        if (bindingsUtils.isPriceIndicatorEnabled(chart.series)) {
+            (button as any).firstChild.style['background-image'] =
+            'url("' + stockTools.getIconsURL() + 'current-price-hide.svg")';
+        } else {
+            (button as any).firstChild.style['background-image'] =
```

Comment:
Nothing big, but you could declare the variable like this
```js
button = chart.stockTools &&
            chart.stockTools.toolbar &&
            (chart.stockTools.toolbar.querySelector('.highcharts-current-price-indicator') as any).firstChild;
```
and use a shorter version
```js
button.style['background-image'] = ...
```